#277 Added SaaS configuration UI (#278)
exasol/ai-lab: doc/changes/changes_2.1.0.md

## Features

> * 277 Added configuration for a SaaS Database. 
>       Added SaaS as the third option in the database selection UI.

Reviewer: ```suggestion
* #277: Added the SaaS database parameters to the configuration page
```
